### ISSUE TITLE
nan loss.

### ISSUE THREAD

**Issue by feifeiwei** [Tung-I/Dual-awareness-Attention-for-Few-shot-Object-Detection#21]
[epoch  1][iter    0/5095] loss: 357.1830, lr: 1.00e-04
                        fg/bg=(13/1011), time cost: 1.175037
                        rpn_cls: 237.8793, rpn_box: 22.8905, rcnn_cls: 87.1240, rcnn_box 9.2892
[epoch  1][iter   20/5095] loss: nan, lr: 1.00e-04
                        fg/bg=(58/966), time cost: 21.597601
                        rpn_cls: 0.6926, rpn_box: 0.0870, rcnn_cls: nan, rcnn_box nan

i want rto train a few-shot detector with DOTA dataset, but nan loss occured during the second iter,

**Comment by Tung-I** [comment]
Seems like a hyperparameter problem. Try to follow the paper to set appropriate hyperparameters.
